Gopkg.toml

```diff
@@ -194,7 +194,7 @@ required = [
 
 [[constraint]]
   name = "github.com/golang/protobuf"
-  version = "v1.3.0"
```

Comment:
Is there a particular reason to do this update here?

Reply:
It did not compile with 1.3.0 on my local machine. But I am actually not sure if its a hard requirement.